pkg/sfu/connectionquality/scorer.go

```diff
@@ -224,17 +227,6 @@ func (q *qualityScorer) UpdateMute(isMuted bool) {
 
 func (q *qualityScorer) addBitrateTransitionAtLocked(bitrate int64, at time.Time) {
 	q.aggregateBitrate.AddSampleAt(bitrate, at)
-
-	if bitrate == 0 {
-		if !q.isLayerMuted() {
-			q.layerMutedAt = at
-			q.score = maxScore
-		}
-	} else {
-		if q.isLayerMuted() {
-			q.layerUnmutedAt = at
-		}
-	}
```

Comment:
Removing this part. This was aimed to make the API easier to use. For example, adding a bitrate transition with non-zero values means the track is active and hence mute should be removed. But, it is getting complicated due to
1. This scorer is used by both down stream and up stream. And they have different meanings. Down stream could be setting 0 bitrate due to congestion pause. Up stream due to dynacast pausing layers.
2. Down stream and up stream behave differently with dynacast/adaptive stream etc. So, combining is not great. 

Switching to explicit signals from callers.